core/os/shell/local.go

```diff
@@ -58,17 +56,17 @@ func (localTarget) Start(cmd Cmd) (Process, error) {
 	// first child time to close-on-exec all of it's fds.
 	// fix source: https://groups.google.com/forum/#!topic/golang-dev/4efaTJ5uA8Y
 	// context: https://golang.org/src/syscall/exec_unix.go?h=StartProcess#L17
-	for {
-		err := p.exec.Start()
+	// for {
```

Comment:
I see you've removed it in a later CL. Good. It probably makes sense to squash temporary commits like this before reviewing.